src/styles/_header.scss

```diff
@@ -1,3 +1,5 @@
+$Smoothly-duration: 0.3s;
```

Comment:
move it to the variables file, and use it in both files
also name it properly `transition-time` or smth